e2e/README.md

```diff
@@ -1,92 +1,62 @@
+An important consideration is that the local developer experience changed when we moved from using Docker containers to real VMs, so the framework is not currently targeted to be run locally, only using the CI (manually or via pull-request). We are currently working on improving that part.
 
-> It's possible to create a new test suite with `SUITE=name make -C e2e create-suite`, which creates the build files and the scaffolding for the first test suite.
+The @elastic/observablt-robots team is maintaining the framework and, because of the current needs on the Fleet team, the test scenarios. On the other hand, our main focus as the Robots team is to only maintain the core functionalities of the e2e test project, and let the teams maintain their scenarios by themselves with our assistance.
 
-### Services as Docker containers
-The services provided by the test suites in this framework will be started in the form of Docker containers. To manage the life cycle of those containers in test time we are going to use [`Testcontainers`](https://golang.testcontainers.org), a set of libraries to simplify the usage of the Docker client, attaching container life cycles to the tests, so whenever the tests finish, the containers will stop in consequence.
+## Contributors and maintenance
+We have received contributions from multiple teams in different aspects of the e2e tests project, so we are ecstatic to receive them:
 
-### Runtime dependencies
-In many cases, we want to store the metrics in Elasticsearch, so at some point we must start up an Elasticsearch instance. Besides that, we want to query the Elasticsearch to perform assertions on the metrics, such as there are no errors, or the field `f.foo` takes the value `bar`. For that reason we need an Elasticsearch instance in a well-known location. We are going to group this Elasticsearch instance, and any other runtime dependencies, under the concept of a `profile`, which is a represented by a `docker-compose.yml` file under the `cli/config/compose/profiles/` and the name of the test suite.
+  - Julia Bardi and Nicolas Chaulet, from Fleet team, frontend engineers, have contributed a few scenarios for Fleet
+  - Eric Davis, QA engineer, has helped in the definition of the scenarios for Fleet
+  - Igor Guz, from Security team, QA engineer, has contributed scenarios for the security-related integrations, such as Endpoint, Linux and System
+  - Christos Markou and Jaime Soriano, from Beats, have contributed and currently maintain the entire k8s-autodiscover test suite
+  - Julien Lind, from Fleet, has helped in defining the support matrix in terms of what OSs and architectures need to be run for Fleet test suite
+  - Julien Mailleret, from Infra, has contributed to the Helm charts test suite.
+  - Anderson Queiroz, from Elastic Agent, is currently working on the MacOS support for running the tests on real Apple machines.
```

Comment:
Eh? I just talked to @adam-stokes yesterday and I was left with the impression that this was still a TODO for him to do and that we were going to need to integrate this with the upcoming Orka integration, where we're already trying to estimate the total number of machines we might need. Could you please link me to an issue or issues where this support is being discussed?

Reply:
I think we can merge this one even without the feedback from orka support